examples/supplychain-app/configuration/roles/helm_component/templates/frontend.tpl

```diff
@@ -21,7 +25,7 @@ spec:
       nodePorts:
         port: {{ peer_frontend_port }}
         targetPort: {{ peer_frontend_targetport }}
-      image: {{ network.container_registry.url | lower }}/bevel-supplychain-frontend:latest
+      image: ghcr.io/hyperledger/bevel-supplychain-frontend:stable
```

Comment:
this should come from network yaml; and the ghcr repo should be added in samples

Reply:
Changed.